blocks/bricks/conv.py

```diff
@@ -335,9 +335,15 @@ def get_dim(self, name):
         if name == 'input_':
             return self.input_dim
         if name == 'output':
-            return tuple(Pool.out_shape(
-                self.input_dim, self.pooling_size, st=self.step,
-                ignore_border=self.ignore_border, padding=self.padding))
+            if None in self.input_dim[-2:]:
```

Comment:
Okay, I've changed the comparison to check against `(None, None)` for now. If you're considering making some larger changes for how this case is handled (like in `Convolutional`, `ConvolutionalSequence`, etc.), I think it'd still be good to have `Pooling` work in this case for the interim.